demo/demo4.html

```diff
@@ -93,13 +97,13 @@
   <script>
 
     // custom transformation: scale header's title
-    var titleStyle = document.querySelector('.title').style;
+    var title = document.querySelector('.title');
     addEventListener('paper-header-transform', function(e) {
       var d = e.detail;
       var m = d.height - d.condensedHeight;
       var scale = Math.max(0.75, (m - d.y) / (m / 0.25)  + 0.75);
-      titleStyle.transform = titleStyle.webkitTransform =
-          'scale(' + scale + ') translateZ(0)';
+
+      Polymer.Base.transform('scale(' + scale + ') translateZ(0)', title);
```

Comment:
We can also use scale3d like this `Polymer.Base.transform('scale3d(' + scale + ',0,0)', title);` and can remove `isolation: isolate` from the above CSS.